### ISSUE TITLE
Select sections/groups

### ISSUE THREAD

**Issue by zampage** [patosai/tree-multiselect.js#59]
Can I somehow select a section/group but not their children?

Like turn of `allowBatchSelection` but then add checkboxes to the sections/groups.

**Comment by patosai** [comment]
Yes, `onlyBatchSelection`.

**Comment by zampage** [comment]
Yes exactly, I want to select only the section! Isn't there somehow a possibilty to do it? Maybe add a custom checkbox for the section after disabling `onlyBatchSelection`.

**Comment by patosai** [comment]
Sorry, there's no method for doing that since there will be no change in the `select` element.

**Comment by itorlakov** [comment]
You can select it kinda rough with this
```
  $("div.section div.title input").on('change', function(){
    alert( 'clicked!' );
  });
```

**Comment by patosai** [comment]
If anyone is still interested in this, what would the value of a 'section' be? This plugin tries not to modify the select DOM node, but if you want to extract a value for a section, there would need to be generated options added to the node.

**Comment by benhaze** [comment]
Still interested! Value could be the same as the nodename/sectionname (maybe with any prefix/suffix or both?!).
Section of this Node is the parent Section.

something like:
x prop1      Props
x prop2      Props/Sec1
x prop3      Props/Sec2/Sec4
x [Sec3]      Props
